modules/tinymce/src/core/main/ts/mode/Readonly.ts

```diff
@@ -94,6 +94,10 @@ const toggleReadOnly = (editor: Editor, readOnlyMode: EditorReadOnlyType): void
     if (Type.isBoolean(readOnlyMode) ? readOnlyMode : !readOnlyMode.selectionEnabled) {
       setContentEditable(body, false);
       switchOffContentEditableTrue(body);
+    } else {
+      if (editor.hasEditableRoot()) {
+        setContentEditable(body, true);
```

Comment:
Why isn't this just a call to `unsetEditorReadonly`?

Reply:
We still want to keep the editor in "readonly" mode. But only allowing the cursor to be placed, so reverting the contenteditable to true.